compiler/testData/diagnostics/tests/thisAndSuper/unqualifiedSuper/unqualifiedSuperWithInterfaces.fir.kt

```diff
@@ -20,9 +20,9 @@ interface AnotherInterface {
 interface DerivedInterface: Interface, AnotherInterface {
     override fun foo() { super.foo() }
     override fun ambiguous() {
-        super.<!UNRESOLVED_REFERENCE!>ambiguous<!>()
+        <!AMBIGUOUS_SUPER!>super<!>.ambiguous()
     }
     override val ambiguousProp: Int
-        get() = super.<!UNRESOLVED_REFERENCE!>ambiguousProp<!>
+        get() = <!AMBIGUOUS_SUPER!>super<!>.ambiguousProp
```

Comment:
I am not sure what you mean. It seems FIR and FE1.0 are both reoprting `AMBIGUOUS_SUPER` in this case.

Reply:
Sorry, I meant the file `testData/diagnostics/tests/thisAndSuper/unqualifiedSuper/unqualifiedSuperWithInnerClass.kt`. It reports `AMBIGUOUS_SUPER` instead of `UNRESOLVED_REFERENCE`. Does this test pass for you?

Reply:
Ah I see. Thanks! Let me fix that.